web/client/components/TOC/TOCItemsSettings.jsx

```diff
@@ -123,7 +124,7 @@ const TOCItemSettings = (props) => {
                         nodeType={settings.nodeType}
                         settings={settings}
                         retrieveLayerData={onRetrieveLayerData}
-                        onChange={(key, event) => isObject(key) ? onUpdateParams(key, realtimeUpdate) : onUpdateParams({[key]: event.target.value, label: event.target.label}, realtimeUpdate)}
```

Comment:
The `onChange` event is passing an object called `event`. The function handled by the event was expecting the event object to having keys and values for `event.target.value `and `event.target.label` but those seem to be non-existent; the added fix handles the function the absence of the aforementioned keys.